ixwebsocket/IXWebSocketTransport.cpp

```diff
@@ -234,6 +231,23 @@ namespace ix
         _onCloseCallback = onCloseCallback;
     }
 
+    void WebSocketTransport::initTimePointsAndGCDAfterConnect()
+    {
+        {
+            std::lock_guard<std::mutex> lock(_lastSendPingTimePointMutex);
+            _lastSendPingTimePoint = std::chrono::steady_clock::now();
+        } 
+        {
+            std::lock_guard<std::mutex> lock(_lastReceivePongTimePointMutex);
+            _lastReceivePongTimePoint = std::chrono::steady_clock::now();
+        }
+
+        if (_pingIntervalOrTimeoutGCDSecs > 0)
+        {
+            _nextGCDTimePoint = std::chrono::steady_clock::now() + std::chrono::seconds(_pingIntervalOrTimeoutGCDSecs);
```

Comment:
The same calculation is in `configure` method. Better to move it to new method to avoid duplication of code

Reply:
I already removed it from the configure method

Reply:
okay